proto/buf/alpha/registry/v1alpha1/download.proto

```diff
@@ -21,20 +21,38 @@ import "buf/alpha/module/v1alpha1/module.proto";
 // DownloadService is the download service.
 service DownloadService {
   // Download downloads.
+  // Soon to be deprecated.
   rpc Download(DownloadRequest) returns (DownloadResponse);
+  // DownloadManifest downloads a module in the manifest+blobs encoding format.
+  rpc DownloadManifest(DownloadManifestRequest) returns (DownloadManifestResponse);
```

Comment:
Do we want to name these `DownloadManifestAndBlobs` to be more accurate?

Reply:
Yes could be, I'm more for shorter, so another option is `*CAS` suffix, but that could be cryptic?

Reply:
I don't mind *CAS.